compiler-rt/test/pfp/CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+set(PFP_TESTSUITES)
+set(PFP_TEST_DEPS
+  ${SANITIZER_COMMON_LIT_TEST_DEPS})
+
+
+macro(add_pfp_testsuite arch lld thinlto compilerrt libunwind)
+  set(PFP_TEST_TARGET_ARCH ${arch})
+  get_test_cc_for_arch(${arch} PFP_TEST_TARGET_CC PFP_TEST_TARGET_CFLAGS)
+
+  string(TOUPPER ${arch} CONFIG_NAME)
+
+  if (${thinlto})
+    set(CONFIG_NAME "thinlto-${CONFIG_NAME}")
+    list(APPEND PFP_TEST_DEPS LTO)
+  endif()
+   if (${libunwind})
```

Comment:
I don't think we need this if, we can just link libunwind unconditionally. Depending on how we end up implementing the x86 side we may need to depend on compiler-rt there as well.